driver/icons.go

```diff
@@ -0,0 +1,21 @@
+package driver
```

Comment:
This interface is just the thing, but interfaces should be at the top level, where sub-packages are free to implement them without the main code having to know where to look